### ISSUE TITLE
Support window.parent object.

### ISSUE THREAD

**Issue by egomezr** [nolimits4web/swiper#3663]
This is a (multiple allowed):
* [ ] bug
* [ ] enhancement
* [x] feature-discussion (RFC)

* Swiper Version: 6.0.0.
* Platform/Target and Browser Versions: macOS, Windows, CORDOVA, IOS, ANDROID, CHROME.
* Live Link or JSFiddle/Codepen or website with issue: N/A.

### What you did
I have an iframe and inside of it I'm creating dynamically the elements for my swiper, these elements will be appended to the parent window document body, and after these elements were appended, I did the following:

```
let $htmlElement = $(Dynamically created element).
$(window.parent.document.body).append($htmlElement);
let swiper = new Swiper($htmlElement[0], {...});
```

### Expected Behavior
Basically, the expected behavior is to work using a specific window object.
**For example, I could pass an option as follow:**

```
{
    windowObject: window.parent
}
```

### Actual Behavior
What I did doesn't work because swiper.js uses the current window **(in this case iframe's)**, so the swiper instance is created, however, the components are inconsistent and wrongly rendered.

**Comment by nolimits4web** [comment]
What, why is it ever needed? Can’t even imagine any library that accept another window object

**Comment by egomezr** [comment]
> What, why is it ever needed? Can’t even imagine any library that accepts another window object

Did you read what I wrote? If you have another way to do it, just let me know.

**Comment by nolimits4web** [comment]
There will no be window parent added as there is too many dependents. Closing as not really related to Swiper but to custom implementation
